### ISSUE TITLE
Cannot recover a archived forum from Freeflarum services website.

### ISSUE THREAD

**Issue by shawny43** [gwillem/freeflarum.com#303]
__Describe the bug:__

A software bug is an error, flaw or fault in a computer program that causes it to produce an incorrect or unexpected result, or to behave in unintended ways.

Try to recover a forum archived from the main FreeFlarum website but there error without additional informations. I follow whats the website said and i can't recover my archived forum. 

__My FreeFlarum forum URL:__

https://myforum.flarum.cloud

**To Reproduce:**

1. Go to reactivate forum
2. Enter informations (Forum+Password)
3. Click reactivate button
4. Show the error.

...etc.

**Expected behavior:**

The forum behaves normally *(obviously try to be more specific about your issue)*.

Normally is supposed to work, but now i can't recover.

**Screenshot(s) of the bug (optional):**
![image](https://user-images.githubusercontent.com/53010077/144521471-545a4c1a-3f76-48e0-9143-be93fd3672e3.png)

**Additional notes:**
I try to recover due i have a project in mind to run a community game server but is not a final decision.

**Comment by CWKevo** [comment]
Are you sure that https://myforum.flarum.cloud/ is your forum?

**Comment by CWKevo** [comment]
I've looked into this and I Sentry actually caught this error. Your forum no longer has an archived copy to reactivate it from because [we purged old archives a while ago to save disk space](). However, there still seems to be a backup of your forum in another folder (which I haven't purged). Shall I reactivate it from there? It might recover your forum.

**Comment by shawny43** [comment]
Soo why when i tried to enter my site name that said there a archived one? Idk whats is going on.

**Comment by shawny43** [comment]
There the name about it. Delatlon.flarum.cloud if you want to know.

**Comment by CWKevo** [comment]
> Soo why when i tried to enter to recreate my new site name that said there a archived one from my old save? Idk whats is going on.

FreeFlarum archives deleted forums in directory called `attic`. We've decided to clean old archives in this directory a while ago (in the discussion that I've linked above). The reactivation feature is only programmed to restore forums from `attic`, so your save was simply not found after the cleanup - therefore, that failed.

However, in case anything goes wrong, we also backup all forums every day at midnight. These backups are named after days in the week (so `Monday`, `Tuesday`, etc...). When next week comes, old backups are overwritten. Currently, backups of deleted forums are not deleted along with them (this behavior is not intended and will be fixed). I've decided to not fix it now, in case people like you would like to reactivate a forum that is not available in `attic`, therefore seemingly destroyed. But in fact, it still lies in another folder. I hope that this makes sense.


> There the name about it. Delatlon.flarum.cloud if you want to know.

Yes, I saw it in Sentry logs. An exception was raised server-side that your forum was not found in `attic`, and Sentry captured it. So, shall I reactivate it from the last known backup? I am asking because I will need to create a new placeholder forum and then overwrite that forum with data from backup, which takes some effort, so I want to make sure whether or not to reactivate it.
